gaphas/canvas.py

```diff
@@ -743,6 +742,7 @@ def update_matrix(self, item, parent=None):
         """
         Update matrices of an item.
         """
+        Matrix = cairo.Matrix
```

Comment:
I think it would be cleaner to just update the other Matrix calls to use cairo.Matrix.